docs/io/configuration/components/models/converters/arepo_to_tardis.ipynb

```diff
@@ -0,0 +1,220 @@
+{
```

Comment:
<p>You should explain what those two files are. One other suggestion: instead of using try/except, you could just put the code in a markdown codeblock and explain that it shows how to take the snapshot if you have the package installed.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/tardis/pull/1867/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>